README.md

```diff
@@ -19,10 +19,10 @@ You can put string or object in an array.
     "literal-blacklist"
   ],
   "rules": {
-    "literal-blacklist/literal-blacklist": [2, ["put", "your", "string", "or", {
+    "literal-blacklist/literal-blacklist": [2, {"ignoreCase": true, literals: ["put", "your", "string", "or", {
       term: "object",
       message: "custom message",
```

Comment:
```suggestion
    "literal-blacklist/literal-blacklist": [2, ["put", "your", "string", "or", {
      term: "object",
      message: "custom message",
      ignoreCase: true,
```

Reply:
I think there should be an `ignoreCase` for each word, since you may want to set it for each word.
It would also keep backward compatibility.

Reply:
yeah, I agree!

Reply:
It's true that there are cases where you need to write `ignoreCase` over and over again.
However, as long as we set the `ignoreCase` for each word, I think it is unavoidable.

Reply:
I think then ignoreCase has no benefit, why won't then I write the same word two times instead of writing ignoreCase with each word, the sole purpose of this proposal was to avoid repetition.

Reply:
I think we should not configure the entire set of rule, as you will have to consider rule that include both case-sensitive and case-insensitive words.

[For example](https://github.com/kyaido/eslint-plugin-literal-blacklist/issues/9), 'Teilnehmer' is case-insensitive, but 'ebay' is case-sensitive.

```
"literal-blacklist/literal-blacklist": [
	"error", [
		{
			"term": "Teilnehmer",
			"message": "we use »Teilnehmende« to be gender-neutral",
		},
		{
			"term": "ebay",
			"message": "The brand is »eBay«, not »ebay« ;)",
		},
	],
],
```

Reply:
If we have a rule with both sensitive and non-sensitive then we can set the entire rule as sensitive and repeat the words for non-sensitive. here default ignoreCase is False.
Example:
"literal-blacklist/literal-blacklist": [
	"error", [
		{
			"term": "Teilnehmer",
			"message": "we use »Teilnehmende« to be gender-neutral",
		},
		{
			"term": "ebay",
			"message": "The brand is »eBay«, not »ebay« ;)",
		},
              {
			"term": "eBay",
			"message": "The brand is »eBay« ;)",
		},
	],
],

Reply:
I think that also makes sense. I have changed the code to make it backwards compatible.

Reply:
@kyaido would you like to review it?

Reply:
Of course.
But sorry, I'm busy and will review it within a few days.

Reply:
@luqmanulkhair 
v1.1.0 has been released with the ignoreCase option added!